configuration/messageproperties/messages_en.properties

```diff
@@ -3076,3 +3076,6 @@ mirebalais.cohortDefinition.diedExitFromLocationDuringPeriodSoonAfterAdmission.n
 mirebalais.cohortDefinition.diedExitFromLocationDuringPeriodSoonAfterAdmission.description = Had an Exit From Inpatient encounter at {{location}} between {{startDate}} and {{endDate}}, for which the previous disposition was Death, AND died within 48 hours of admission
 mirebalais.cohortDefinition.diedExitFromLocationDuringPeriodNotSoonAfterAdmission.name = Died in Inpatient Ward during period, at least 48h after admission
 mirebalais.cohortDefinition.diedExitFromLocationDuringPeriodNotSoonAfterAdmission.description = Had an Exit From Inpatient encounter at {{location}} between {{startDate}} and {{endDate}}, for which the previous disposition was Death, AND did not die within 48 hours of admission
+
+# override a property defined in Core Apps
```

Comment:
So this overrides the property defined in Core Apps... I'm not sure if that guaranteed/defined functionality in Iniz, but it seems nice.

As @mseaton  suggested I think we could make a "Haiti-only" file if we only want to make this change for Haiti, but that might be overkill, I don't think other sites will care too much... maybe with the exception of Peru and Mexico, but then @brandones  you might just want to provide a Spanish translation?